presto-hive/src/main/java/com/facebook/presto/hive/security/ranger/RangerBasedAccessControlModule.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.facebook.presto.hive.security.ranger;
+
+import com.facebook.airlift.configuration.AbstractConfigurationAwareModule;
+import com.facebook.presto.spi.connector.ConnectorAccessControl;
+import com.google.inject.Binder;
+import com.google.inject.Scopes;
+
+import static com.facebook.airlift.http.client.HttpClientBinder.httpClientBinder;
+
+public class RangerBasedAccessControlModule
+        extends AbstractConfigurationAwareModule
+{
+    @Override
+    public void setup(Binder binder)
+    {
+        binder.bind(ConnectorAccessControl.class).to(RangerBasedAccessControl.class).in(Scopes.SINGLETON);
+        RangerBasedAccessControlConfig rangerBasedAccessControlConfig = buildConfigObject(RangerBasedAccessControlConfig.class);
+
+        httpClientBinder(binder).bindHttpClient("rangerInfo", ForRangerInfo.class)
+                .withConfigDefaults(
+                        config -> config.setAuthenticationEnabled(false)
+                ).withFilter(RangerBasicAuthHttpRequestFilter.class)
+                .withConfigDefaults(config -> {
+                    config.setKeyStorePath(rangerBasedAccessControlConfig.getRangerRestKeystorePath());
+                    config.setKeyStorePassword(rangerBasedAccessControlConfig.getRangerRestKeystorePwd());
+                    config.setTrustStorePath(rangerBasedAccessControlConfig.getRangerRestTruststorePath());
+                    config.setTrustStorePassword(rangerBasedAccessControlConfig.getRangerRestTruststorePwd());
+                });
```

Comment:
I missed this. I was taking reference of http client used in Druid connector, didn't find any document about extra config flexibility which is coming with airlift's http client so. I will take out Keystore & Truststore related config from Ranger config properties since they will not be needed anymore.